.eslintrc

```diff
@@ -8,6 +8,7 @@
     "plugin:prettier/recommended",
     "prettier"
   ],
+  "ignorePatterns": ["test/specs/*"],
```

Comment:
Fixing tests and refactoring to .tsx files are in progress. Idk if we actually want to ignore the errors we're trying to fix. Might just ask you do remove this for now so we can address it elsewhere

Reply:
of course! I hadn't appreciated these upcoming changes, so l've reverted this change